index.html

```diff
@@ -15,12 +16,17 @@
     <header>
         <nav class="navbar">
             <a href="#" class="logo">My Logo</a>
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
             <i id="menu" class="fa-solid fa-bars"></i>
         </nav>
     </header>
     <section class="headline">
```

Comment:
- [ ] I suggest using the right alignment in the desktop version in order to match the Figma design 

- [ ] Also please try to use the right illustration background for desktop version as you can see the illustration not clean in the desktop version

- Yours 
![image](https://user-images.githubusercontent.com/61790587/177779885-9042b422-dbb4-41e8-b16c-e8d19896b6d0.png)
- Figma
![image](https://user-images.githubusercontent.com/61790587/177779905-1ef21d07-cab4-4797-b7d4-60baf916e211.png)